run-build.sh

```diff
@@ -93,11 +93,18 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --env-vars)
```

Comment:
Can we support this option on `run-build.ps1` as well?  That way our Windows and Unix builds stay in lock step.